terraform/helm/aptos-node/templates/validator.yaml

```diff
@@ -111,7 +111,9 @@ spec:
         - containerPort: 6180
         - containerPort: 6181  # VFN
         - containerPort: 6182  # Public
+  {{- if $.Values.service.validator.enableRestApi }}
```

Comment:
we don't need this change here, as the pod will not be routed to since port 8080 would be blocked on the k8s service: https://github.com/aptos-labs/aptos-core/pull/5285/files#diff-53c6da200db284fe6f0633e59b0dcf684dcf7dd9f5b41ccc4b8e1f7bd86aae21R22-R24